.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,43 @@
+name: 'Testing XamlX'
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest ]
+
+    name: Build ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    steps:
+
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.402
+
+    - name: Build CecilNetstandardTests
+      run: dotnet build tests/CecilNetstandardTests/CecilNetstandardTests.csproj
+    - name: Build CecilTests
+      run: dotnet build tests/CecilTests/CecilTests.csproj
+
+    - name: Test for netcoreapp3.1
+      run: dotnet test --framework netcoreapp3.1
+
+    - name: Test for net47 CecilNetstandardTests
+      if: matrix.os == 'windows-latest'
+      run: dotnet test --framework net47 tests/CecilNetstandardTests/CecilNetstandardTests.csproj
+    - name: Test for net47 CecilTests
+      if: matrix.os == 'windows-latest'
+      run: dotnet test --framework net47 tests/CecilTests/CecilTests.csproj
+    - name: Test for net47 XamlParserTests
+      if: matrix.os == 'windows-latest'
+      run: dotnet test --framework net47 tests/XamlParserTests/XamlParserTests.csproj
```

Comment:
See also https://github.com/kekekeks/XamlX/pull/35#discussion_r508069535

Probably the following setup might just work:

```yaml
- name: Build Unit Test Projects
  run: dotnet build tests/**/*Tests.csproj
- name: Run Unit Tests
  run: dotnet test --no-build /p:Include="*" /p:Exclude="[*Tests.*]*"
  working-directory: src
```